pages/home/home.html

```diff
@@ -0,0 +1,255 @@
+<section id="main-content">
+    <md-card layout-fill>
```

Comment:
Deindent this and all following lines by 2. Indentation in HTML should be by 2 spaces.

Reply:
Done!

Reply:
Not done?

Reply:
Didn't actually see the <md-card> tags. My bad there.